### PR TITLE
feat: cartList item displayed and remove, move feature added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this PR, Features which are added are - 
1. Cart items that are added are shown.
2. Add to cart.
3. Add to wishList.
4. Remove from cart.
5. Remove from wishlist.
please review and give feedback for imporvement.

### CODE REVIEW

File: src/components/wishCard/wishCard.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    return (
+        <>
+            {
+                wishList.map(wishItem => {
+                    return (
+                        <div className="product--card " key={wishItem.id}>
+                            <img src={wishItem.image} alt="swift car" className="product__img" />
+                            <h3 className="product--name">{wishItem.name} </h3>
+                            <p className="product--version">{wishItem.version} </p>
```
Comment (mariyasada) [inline src/components/wishCard/wishCard.jsx:18]: you can destructure wishitem here

**Review by mariyasada — APPROVED** [review]
good to go

**Review by SaurabhChawda — APPROVED** [review]
You can merge it.

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

**Pull request merged**